cmd/dagger/main.go

```diff
@@ -21,16 +21,17 @@ import (
 )
 
 var (
+	cpuprofile = os.Getenv("CPUPROFILE")
+	pprofAddr  = os.Getenv("PPROF")
```

Comment:
Pardon the unrelated changes, but I noticed at one point that these flags don't seem to work, and then we hid them, so now I just made them env vars.